### PR TITLE
Remove style check from Azure and add to GH Actions

### LINKED ISSUE #497: Easier yapf checking
We currently only do style checking on Azure, which is slower than GH Actions and requires several extra clicks to see what is going on. Ideally we could move this step over to GH Actions and integrate it with our local development process so we spend less time tracking down style issues during PR review.

@henrykironde - is this something we can add to pytest or add a git hook for for local development? Can you try setting up the current test running on Azure on GH Actions?

### PR DESCRIPTION
**Pull request by satsin06**
Fix #497

### CODE REVIEW

**Comment by satsin06** [comment]
Hi, can we add Hacktoberfest topic to this repository?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -28,13 +28,7 @@ jobs:
...
       pip install .
     displayName: 'Create Conda environment for DeepForest'
-  
-  - script: |
-      pip install pytest pytest-azurepipelines yapf
-      pytest
-      yapf -d --recursive deepforest/ --style=.style.yapf 2>&1
-    displayName: 'Pytests and sytle tests'
-    
+
```
Comment (henrykironde) [inline azure-pipelines.yml:31]: You don't have to remove the pytest
```
  - script: |
      pip install pytest pytest-azurepipelines yapf
      pytest -v
    displayName: 'Pytests'
```

**Review by henrykironde — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (satsin06) [reply azure-pipelines.yml:31]: I have resolved this change

File: azure-pipelines.yml [hunk 2]
```diff
@@ -28,13 +28,12 @@ jobs:
       python -m pip install --upgrade pip
       pip install .
     displayName: 'Create Conda environment for DeepForest'
-  
+
   - script: |
       pip install pytest pytest-azurepipelines yapf
       pytest
```
Comment (henrykironde) [inline azure-pipelines.yml:34]: Can you add a v(verbose) to the pytests?
`pytest -v`

Comment (satsin06) [reply azure-pipelines.yml:34]: Okay

**Comment by henrykironde** [comment]
Add yapf to dev _requirements too

**Review by henrykironde — APPROVED** [review]

**Comment by henrykironde** [comment]
Thanks @satsin06

**Pull request merged**